src/FixedDataTableCell.js

```diff
@@ -88,11 +89,20 @@ var FixedDataTableCell = React.createClass({
     };
   },
 
-  shouldComponentUpdate(nextProps) {
-    return (
-      !nextProps.isScrolling ||
-      this.props.rowIndex !== nextProps.rowIndex
-    );
+  shouldComponentUpdate(newProps) {
+    if (newProps.isScrolling && this.props.rowIndex === newProps.rowIndex) {
+      return false;
+    }
+
+    if (!shallowEqual(this.props, newProps, ['cell', 'isScrolling'])) {
+      return true;
+    }
+
+    if (!shallowEqual(this.props.cell.props, newProps.cell.props)) {
```

Comment:
This probably needs some additional work for other consumers. Maybe we need to check state as well or something